components/atomic/Tile/patterns/InvestigationTile/styles.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable */
 import styled from "styled-components";
-import BaseMixedLink from "@/atomic/MixedLink";
+import { MixedLink as BaseMixedLink } from "@rubin-epo/epo-react-lib";
```

Comment:
I'd like to point this back to atomic/index.js like the others but for some reason aliasing that export as `BaseMixedLink` produced `undefined` but aliasing the export from `@rubin-epo/epo-react-lib` worked